modules/claims_api/catalog-info-v2.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: backstage.io/v1alpha1
+kind: API
+metadata:
+  name: benefits-claims-v2
+  description: The Benefits Claims API is a suite of services that enables VA developers to build applications that let Veterans view and submit certain claims and claim-related information to the VA. We provide the ability to auto-establish claims by submitting structured data that can return realtime validations, allowing for better data quality.
+  annotations:
+    github.com/project-slug: https://github.com/department-of-veterans-affairs/vets-api
+    datadog/production-monitor-id: '98a-nx5-icf'
+    datadog/sandbox-monitor-id: '2ne-mbd-mve' # status tab
+    hub.lighthouse.va.gov/api-id: 'claims' # release notes tab
```

Comment:
do these need to be unique? or is it supposed to be labeled under the same one to show they're just different versions?

Reply:
Are you referring to the release notes? if so, idk, and was wondering that myself. So, I was going to reach out to Giles again to ask.